packages/relay/src/lib/precheck.ts

```diff
@@ -56,16 +57,6 @@ export class Precheck {
     return typeof transaction === 'string' ? Transaction.from(transaction) : transaction;
   }
 
-  /**
-   * Checks if the value of the transaction is valid.
-   * @param {Transaction} tx - The transaction.
-   */
-  value(tx: Transaction): void {
-    if (tx.data === EthImpl.emptyHex && tx.value < constants.TINYBAR_TO_WEIBAR_COEF) {
-      throw predefined.VALUE_TOO_LOW;
-    }
-  }
```

Comment:
I think we still want to check that the `tx.value` is not negative, instead of completely removing the pre-check